src/Data/Graph/Acyclic.agda

```diff
@@ -15,10 +15,10 @@ module Data.Graph.Acyclic where
 open import Level using (_⊔_)
 open import Data.Nat.Base as ℕ using (ℕ; zero; suc; _<′_)
 open import Data.Nat.Induction using (<′-rec; <′-Rec)
-import Data.Nat.Properties as ℕ
+import Data.Nat.Properties as ℕ using (≤⇒≤′)
 open import Data.Fin as Fin
   using (Fin; Fin′; zero; suc; #_; toℕ; _≟_; opposite) renaming (_ℕ-ℕ_ to _-_)
```

Comment:
I'd put the renaming directive on another newline; with same indentation as the `using` directive above it...
```suggestion
  using (Fin; Fin′; zero; suc; #_; toℕ; _≟_; opposite)
  renaming (_ℕ-ℕ_ to _-_)
```